docs/REST.md

```diff
@@ -0,0 +1,51 @@
+# REST interfaces
+
+## Repair scheduler
+
+The REST interface for the repair scheduler is located under the path `<host>/repair-scheduler/v1/`.
+The following sub-paths exists:
+* `/get/<keyspace>/<table>`
+* `/list[/<keyspace>]`
+
+The interface is only exposing state for scheduled tables.
+
+
+### Get
```

Comment:
It is a little confusing that to call this get as you are doing GET requests whether you are doing list or get. Maybe remove the titles Get and List - And just list it from a 'What can be done' perspective.

Reply:
Done